coffeelint.json

```diff
@@ -0,0 +1,19 @@
+{
+    "check_scope": {
+      "module": "coffeescope2",
+      "level": "warn",
+      "environments": [],
+      "globals": {},
+      "overwrite": false,
+      "same_scope": false,
+      "shadow": false,
+      "shadow_builtins": false,
+      "shadow_exceptions": [],
+      "undefined": false,
+      "hoist_local": true,
+      "hoist_parent": true,
+      "unused_variables": true,
+      "unused_arguments": false,
+      "unused_classes": false
```

Comment:
Is this a setting that we could/should add by default to `resin-lint`? So that we can keep custom `coffeelint.json` files to a minimum for our projects